iib/workers/tasks/utils.py

```diff
@@ -91,11 +91,19 @@ def add_max_ocp_version_property(resolved_bundles, temp_dir):
         return
 
     # Filter index image bundles to get pull spec for bundles in the request
```

Comment:
I believe it's good to change a bit this comment to reflect that it not only filters the index image bundles to get the pull spec but also it get unique ones.

Maybe something like:
```
# Filter index image bundles to get pull spec for unique bundles in the request
``` 

Anyway, it's a suggestion :smiley: